### PR TITLE
Add isMember & memberIcon to returned data

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
isMember will return true if the user is a paid member of the uploader's channel
memberIcon will return the url of the image that appears beside a member's name (if the user is a member)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ The data is returned as an object with a list of comment objects and a continuat
       isVerified: Boolean,
       isOfficialArtist: Boolean,
       hasOwnerReplied: Boolean, // If the video channel replied to the comment
+      isMember: Boolean, // Whether the user that nade the comment is a paid member or not
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline README.md:80]: made*

File: README.md [hunk 2]
```diff
@@ -129,6 +131,8 @@ The data is returned as a list of objects (seen below).
     isVerified: Boolean,
     isOfficialArtist: Boolean,
     hasOwnerReplied: false,
+    isMember: Boolean, // Whether the user that nade the comment is a paid member or not
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline README.md:134]: Same here

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -129,6 +131,8 @@ The data is returned as a list of objects (seen below).
     isVerified: Boolean,
     isOfficialArtist: Boolean,
     hasOwnerReplied: false,
+    isMember: Boolean, // Whether the user that made the comment is a paid member or not
+    memberIcon: String | null // URL of the member icon
```
Comment (PikachuEXE) [inline README.md:135]: I imagine `memberIcon` would be something like:
```js
memberIcon: {
  url: "URL",
  height: 100,
  width: 100,
  otherMetadata: "whatever",
}
```
Actually if you look at how you fetch the URL
`customBadge.thumbnails` is an array of objects not strings
```suggestion
    memberIconURL: String | null // URL of the member icon
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
https://github.com/FreeTubeApp/yt-comment-scraper/pull/27#issuecomment-926050358

**Review by PikachuEXE — APPROVED** [review]

**Comment by PikachuEXE** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc updated

**Comment by peepopoggers** [comment]
Is Url more standard than URL?

**Comment by PikachuEXE** [comment]
See `authorThumb`
There is already `url`
I think it's fine to use `Url` as **suffix**
If we insist on using `URL` in code the naming might look strange sometimes

**Review by peepo5 — APPROVED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Pull request merged**